customChatWidget/README.md

```diff
@@ -183,6 +183,40 @@ Below steps explain how Custom Chat Widget works on a web page
 
 - For testing, run `npm run dev-build` to build a dev package using Babel and `webpack.dev.js` and saves the built minified file into the `public` folder with name `ACChat.js`. The dev version will have console logs.
 
+### Enabling rich messaging
+
+Amazon Connect Chat now allows your agents and customers to use rich text formatting when composing a message, enabling them to quickly add emphasis and structure to messages, improving comprehension. The available formatting options include bold, italics, hyperlinks, bulleted lists, and numbered lists. [Documentation](https://docs.aws.amazon.com/connect/latest/adminguide/enable-text-formatting-chat.html)
+
+1. To enable rich messaging, include the new param when invoking `initiateChat`:
```

Comment:
Can we provide the link to this function for customers using this repo?

Reply:
Added file path to `customChatWidget/../index.js` file. This comment is not referring to chatInterface file correct?